src/js/components/Index.js

```diff
@@ -20,10 +26,13 @@ const VIEW_COMPONENT = {
 
 export default class Index extends Component {
 
-  constructor () {
-    super();
+  constructor (props) {
```

Comment:
@alansouzati I'm not familiar with context - does it needed to be passed to `super` as well?
```js
constructor (props, context) {
  super(props, context);
  ...
}
```